tools/class-parse/Program.cs

```diff
@@ -39,6 +40,9 @@ public static void Main (string[] args)
 				{ "docspath=",
 				  "Documentation {PATH} for parameter fixup",
 				  doc => docsPaths.Add (doc) },
+				{ "parameters-description=",
```

Comment:
Why a new `class-parse --parameters-description=FILE` option, instead of reusing `class-parse --docspath`? They're both related to parameter name fixups.

Also, PR #202 has examples of unit testing `ClassParse.DocumentationPaths`.

Reply:
Not only because there is no reason to mess existing PUBLIC arguments with INTERNAL ONLY feature, it is NOT A DOCS PATH. Whatever feature that uses this feature for NON-docs path is WRONG.

Reply:
We have allowed non-"documentation" paths to be used with `class-parse --docspath` since PR #97 which you -- [at the time](https://github.com/xamarin/Java.Interop/pull/97#issuecomment-256434210) -- thought was acceptable.

The fundamental purpose to `class-parse --docspath` is to provide parameter name information for methods.

The fundamental purpose of *this PR* is to provide parameter name information for methods.

That sounds ~identical to me, and I don't see why we need a completely different abstraction in the form of `JavaParameterNamesLoader` when we have an *existing* abstraction in `IAndroidDocScraper.GetParameterNames()`. (Sure, the *name* is misleading -- "scraping" documentation isn't required -- but the *semantics* are the same: provide method parameter names. Rename `IAndroidDocScraper` to `IJavaMethodParameterNameProvider` and the name is no longer misleading.)

Reply:
It is not preferred but acceptable. Asking for pointless change is not the same.